apps/web/src/components/NftSelector/NftSelectorPreviewAsset.tsx

```diff
@@ -9,9 +9,13 @@ import transitions from '../core/transitions';
 
 type NftSelectorPreviewAssetProps = {
   tokenRef: NftSelectorPreviewAssetFragment$key;
+  resizeToSquare?: boolean;
```

Comment:
this prop was introduced so that the styling in other instances of NftSelectorPreviewAsset is unchanged such as here before conditionally rendering based on `resizeToSquare` (before vs after attached):
<img width="344" alt="Screenshot 2023-10-02 at 1 33 29 PM" src="https://github.com/gallery-so/gallery/assets/49758803/3096dc5a-79a5-41b1-906b-76393b8d670a">  ---  <img width="268" alt="Screenshot 2023-10-02 at 1 45 03 PM" src="https://github.com/gallery-so/gallery/assets/49758803/52b1b6ea-3154-4b26-b7f9-469574819242">